scripts/error-codes/__tests__/__snapshots__/transform-error-messages.js.snap

```diff
@@ -94,3 +94,14 @@ import invariant from 'shared/invariant';
   }
 })();"
 `;
+
+exports[`error transform should support noMinify option 1`] = `
+"import _ReactError from 'shared/ReactError';
+
+import invariant from 'shared/invariant';
```

Comment:
Is this supposed to remove the import or does some other dead code elimination step do that?

Reply:
Yeah that happens in a subsequent step of the Rollup script